.github/workflows/build.yml

```diff
@@ -15,7 +15,7 @@ jobs:
              compiler: gcc
     env:
       CC: ${{ matrix.compiler }}
-      MACOSX_DEPLOYMENT_TARGET: "10.10"
+      MACOSX_DEPLOYMENT_TARGET: "10.12"
```

Comment:
that sounds good

can you also update the README file with the list of platforms ?